jobs/create-group-version-branch.js

```diff
@@ -158,6 +158,10 @@ module.exports = async function (
       // get version for each dependency
       const npmDoc = await npm.get(isFromHook ? `${installationId}:${depName}` : depName)
       const latestDependencyVersion = npmDoc['distTags']['latest']
+      if (!semver.valid(latestDependencyVersion)) {
+        log.warn(`exited transform creation: ${depName} ${latestDependencyVersion} is not a valid version`)
```

Comment:
maybe `latestDependencyVersion: ` prefix like below, so reading the logs is easier?